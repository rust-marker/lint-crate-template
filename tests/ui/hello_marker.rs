fn something() {
    
}

fn main() {
    println!("Hello, World!s");
}
