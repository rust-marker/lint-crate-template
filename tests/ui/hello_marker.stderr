warning: hello, main (From Marker)
 --> $DIR/hello_marker.rs:5:1
  |
5 | / fn main() {
6 | |     println!("Hello, World!s");
7 | | }
  | |_^
  |
  = note: `#[warn(marker::my_lint)]` on by default

warning: 1 warning emitted

